components/Tabs/Tabs.js

```diff
@@ -1,72 +1,43 @@
+
 class TabLink {
   constructor(tabElement){
-    // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
-    // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
-    
-    // We need to find out if a user clicked 'all' cards or a specific category.  Follow the instructions below to accomplish this task:    
-    
-    /* <- Delete this comment block when you work on the if statement
-    // Check to see if this.tabData is equal to 'all'
-    if(){
+    this.tabElement = tabElement;
+    this.tabData = this.tabElement.dataset.tab;
+    if(this.tabData === 'all'){
```

Comment:
Way to go using that If Statement! Not everybody figured that out. Very Convenient